lib/harmonium.ex

```diff
@@ -196,7 +196,14 @@ defmodule Harmonium do
   Renders an input label.
   """
   def rev_label(modifiers, do: block) do
-    content_tag :label, class: rev_label_class(modifiers) do
+
+    for_attr = modifiers
+      |> Keyword.get(:for)
+
+    class = modifiers
+      |> Keyword.take(:class)
```

Comment:
This would result in a breaking change, which isn't ideal. It might be better to create a new `rev_label/4` instead and have it auto-generate the `for` attribute for us. This would follow the same convention for the input fields so that it'd populate it with a matching `id`.

```elixir
def rev_label(f, key, modifiers, do: block) do
  label(f, key, class: rev_label_class(modifiers)) do
    block
  end
end
```

Reply:
thanks -- didn't realize label() generated `for` attributes. So your thing works out of the box, and I just modified the `single_checkbox` call to hit `rev_label/4` instead, commit below.